packages/widget/src/controllers/wallet-manager/manager.ts

```diff
@@ -7,15 +7,40 @@ import Onboard from '@web3-onboard/core';
 import injectedModule from '@web3-onboard/injected-wallets';
 import walletConnectModule from '@web3-onboard/walletconnect';
 import type { ReactiveController, ReactiveElement } from 'lit';
+import type { WalletInit } from '@web3-onboard/common';
 
 import { utils } from 'ethers';
 import { WalletUpdateEvent, walletContext } from '../../context';
 
 export class WalletController implements ReactiveController {
+  readonly WALLET_CONNECT_PROJECT_ID = '2f5a3439ef861e2a3959d85afcd32d06';
+
   host: ReactiveElement;
 
   walletContext: ContextConsumer<typeof walletContext, ReactiveElement>;
 
+  /**
+   * Provides list of wallets specified by 3rd party
+   * if not, provides default list
+   * @param {{ dappUrl?: string }} options
+   * @returns {WalletInit[]}
+   */
+  getWallets(options?: { dappUrl?: string }): WalletInit[] {
+    const specifiedWallets = this.walletContext.value?.wallets;
+
+    if (!!specifiedWallets && specifiedWallets.length > 0)
```

Comment:
nit: could you put at the same line the`return specifiedWallets`? if not, we could open a block.

Reply:
shouldn't linter with prettier take care of this?